index.php

```diff
@@ -1,3 +1,31 @@
 <?php
+use DinoApp\Dinosaur\Dinosaur;
+use DinoApp\Museum\Museum;
+use DinoApp\DinosaurHydrator\DinosaurHydrator;
 
 require_once 'vendor/autoload.php';
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
Missing responsive meta tag means responsiveness doesn't work right now, so I'll need to retest once you've added it!